app/views/static_pages/privacy_policy.html.erb

```diff
@@ -11,8 +11,8 @@
     <p class="govuk-body">
       This work is being carried out by the Department for Education (DfE) Digital, which is a part of DfE. DfE engages the
       private company dxw to help improve and provide the service. For the purpose of data protection legislation, DfE
-      is the data controller for the personal data processed as part of eachers: claim back your student loan repayments.
-      eachers: claim back your student loan repayments is a free and optional service eligible teachers to claim back
+      is the data controller for the personal data processed as part of Claim additional payments for teaching.
+      Claim additional payments for teaching is a free and optional service for eligible teachers to claim back
       student loan repayments.
```

Comment:
This is one of the bits that will change with M&P.

Reply:
Yeah when we allow them to claim M&P payments I think we should add that bit in.